www/components/Button.tsx

```diff
@@ -13,20 +13,28 @@ const Button = (props: Props) => {
 
   const colorClass =
     type === 'primary'
-      ? 'px-3 py-2 shadow-sm  border border-transparent text-white bg-brand-700 hover:bg-brand-800 focus:ring-2 focus:ring-offset-2 focus:ring-brand-500'
+      ? 'px-3 py-2 shadow-sm border border-transparent text-white bg-brand-700 hover:bg-brand-800 focus:ring-2 focus:ring-offset-2 focus:ring-brand-500'
       : 'text-brand-700 bg-none'
 
   const textClass = type === 'primary' ? 'font-medium left-3 group-hover:left-0' : 'font-normal'
 
   const arrowClass = type === 'primary' ? '' : 'relative -left-1 group-hover:left-0'
 
+  let buttonStyles = {
+    textShadow: 'none',
+  }
+  if (type === 'primary') {
+    buttonStyles.textShadow = '0px 0px 6px rgba(13, 128, 86, 0.8)'
+  }
+
   const renderButton = () => (
     <button
       type="button"
       className={`
         inline-flex items-center text-sm leading-4 rounded-md
         focus:outline-none group transition ${colorClass} ${className}
       `}
+      style={buttonStyles}
```

Comment:
I would of suggested not using inline styles but this is going to get replaced with sb-ui anyway.

I'll add a text shadow to the new button component

Reply:
Yeah I'm only treating this as a patch since it's going to get replaced by sb-ui.

Reply:
Thanks for the approval - will merge to close the other issue